kythe/go/platform/tools/kzip/createcmd/createcmd.go

```diff
@@ -211,6 +211,7 @@ func (cb *compilationBuilder) addFile(ctx context.Context, root string) ([]strin
 				Digest: digest,
 			},
 		})
+		files = append(files, path)
```

Comment:
Is there a unit test that could easily be added/updated to catch this?

Reply:
Not easily in this commit since it's a subcommand working on the filesystem.